administration/src/bp-modules/regions/RegionController.tsx

```diff
@@ -20,19 +20,24 @@ const RegionSettingsContainer = styled.div`
 `
 
 const RegionController = (): ReactElement => {
-  const { region, role } = useWhoAmI().me
+  const { region } = useWhoAmI().me
   const { freinetDataTransferEnabled, projectId } = useContext(ProjectConfigContext)
   const { t } = useTranslation('errors')
-  if (!region || role !== Role.RegionAdmin) {
-    return <NonIdealState icon='cross' title={t('notAuthorized')} description={t('notAuthorizedForRegionSettings')} />
-  }
+
   return (
-    <RegionSettingsContainer>
-      <DataPrivacyCard />
-      <RegionSettingsController regionId={region.id} />
-      <ApplicationConfirmationNoteController regionId={region.id} />
-      {freinetDataTransferEnabled && <FreinetSettingsController regionId={region.id} project={projectId} />}
-    </RegionSettingsContainer>
+    <RenderGuard
+      allowedRoles={[Role.RegionAdmin]}
+      condition={region !== undefined}
+      error={{ description: t('notAuthorizedForRegionSettings') }}>
+      {region && (
```

Comment:
Why are we checking `region` twice at condition and here? does condition is sufficient?

Reply:
Because i wanted to avoid `!` you can check what happens without.
Thats the drawback of this component instead of an `if else` statement